memberships/views.py

```diff
@@ -181,23 +181,7 @@ def stripe_webhook(request):
             json.loads(request.body), settings.STRIPE_SECRET_KEY
         )
         stripe_webhook = StripeWebhook()
-        if event["type"] == "invoice.payment_failed":
-            f_payment = FailedPayment.objects.create(
-                stripe_customer_id=event["data"]["object"]["customer"],
-                stripe_subscription_id=event["data"]["object"]["subscription"],
-                stripe_event_type=event["type"],
-            )
-        if event["type"] == "invoice.payment_succeeded":
-            member = Member.objects.get(email=event["data"]["object"]["customer_email"])
-            payment = Payment.objects.create(
-                member=member,
-                stripe_subscription_id=event["data"]["object"]["subscription"],
-            )
-
-            # Store payment DateTime in membership model
-            membership = Membership.objects.get(member=member)
-            membership.last_payment_time = epoch_to_datetime(event["created"])
-            membership.save()
+        handle_stripe_payment(event)
```

Comment:
As per my previous comment, as this method should return an `HttpResponse` you will want to return the result of this function, much like the `stripe_webhook.hanlde()` call below.

```suggestion
        return handle_stripe_payment(event)
```

Reply:
👌

Reply:
Doing this would mean that I can't `return stripe_webhook.handle(event)` below it. (line 185)

Does this mean that I might need to move `StripeWebhook` class into payments.py now?

Reply:
Yeah I think it does, is that something you are happy to do?

Reply:
Yep - I shall start experimenting soon